tests/python/driver/tvmc/test_target_options.py

```diff
@@ -86,6 +86,19 @@ def test_default_arg_for_mrvl_hybrid():
     assert parsed.target_mrvl_num_tiles == 8
 
 
+@tvm.testing.requires_mrvl
+# Test for default(LLVM) target, when built with USE_MRVL=ON
+def test_mrvl_build_with_llvm_only_target():
+    parser = argparse.ArgumentParser()
+    generate_target_args(parser)
+    parsed, _ = parser.parse_known_args(
+        [
+            "--target=llvm",
+        ]
+    )
+    assert parsed.target == "llvm"
```

Comment:
I'm trying to understand what would be expected from this test case if TVM is not built with `USE_MRVL=ON`?

Reply:
The test will be skipped by the decorator when USE_MRVL=OFF i.e @tvm.testing.requires_mrvl